src/components/ProblemFilter/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "all": "All",
+  "with-problems": "With problems"
```

Comment:
i18n namings should meet this ruleset https://github.com/ydb-platform/ydb-embedded-ui/blob/main/i18n-naming-ruleset.md